Login.py

```diff
@@ -0,0 +1,111 @@
+import flask
+import os
+from flask_sqlalchemy import SQLAlchemy
+from dotenv import load_dotenv, find_dotenv
+
+load_dotenv(find_dotenv())
+
+from flask_login import (
+    LoginManager,
+    UserMixin,
+    current_user,
+    login_required,
+    login_user,
+    logout_user,
+)
+
+app = flask.Flask(__name__)
+app.secret_key = os.getenv("app.secret_key")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+db = SQLAlchemy(app)
+
+login_manager = LoginManager()
+login_manager.init_app(app)
+
+
+class User(UserMixin, db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(100), unique=True)
+    password = db.Column(db.String(16))
+
+
+db.create_all()
+
+
+@app.route("/")
+def login():
+    return flask.render_template("login.html")
```

Comment:
where is the login.html and signup.html? I am unfamiliar with how pull requests, but I don't see it in this pull request.